rust/src/lib/ip.rs

```diff
@@ -200,6 +200,40 @@ impl InterfaceIpv4 {
         Self::default()
     }
 
+    /// Set explicit default values if not desired. Used when do not want to
+    /// merge ip settings from current status
+    pub fn set_explicit_defaults(&mut self) {
```

Comment:
I think that the name should be `set_implicit_defaults`, actually, isn't it? Some defaults are set implicitly depending on other values, they are not explicitly specified...

Reply:
Removed. New approach been used.